gno.land/cmd/gnoland/genesis_generate.go

```diff
@@ -20,6 +20,7 @@ type generateCfg struct {
 	blockMaxDataBytes int64
 	blockMaxGas       int64
 	blockTimeIota     int64
+	gnotUnrestricted  bool
```

Comment:
Could it be better to replace `gnot` with `ugnot` in code for accuracy?

Reply:
I can. Keep in mind this is still marked as draft and I plan on changing some things.